easybuild/easyconfigs/p/pbcopper/pbcopper-2.0.0-GCCcore-11.2.0.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'MesonNinja'
+
+name = 'pbcopper'
+version = '2.0.0'
+
+homepage = 'https://github.com/PacificBiosciences/pbcopper'
+description = """The pbcopper library provides a suite of data structures, algorithms,
+ and utilities for C++ applications."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'cstd': 'c++17'}
+
+source_urls = ['https://github.com/PacificBiosciences/pbcopper/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9edbef2fc6e3885c1206b6a9cdf7287bfa826b2b758992526d2f4d3c9b57431e']
+
+builddependencies = [
+    ('Meson', '0.58.2'),
+    ('Ninja', '1.10.2'),
+]
+
+dependencies = [
+    ('Boost', '1.77.0', '', ('GCC', '11.2.0')),
+]
+
+sanity_check_paths = {
+    'files': ['lib/libpbcopper.%s' % SHLIB_EXT],
+    'dirs': ['include/pbcopper', 'lib/pkgconfig'],
+}
+
+moduleclass = 'lib'
```

there should be a newline at the end of the file
```suggestion
moduleclass = 'lib'
```

---

the newline is not accepted by github, nor does it constitute a change?

---

hmm, weird, putting it in a suggestion has worked for me before.

---

@cmeesters guess you'll have to make the change manually. In any case, it's probably worth configuring your editor of choice to not strip newlines from `*.eb` files